slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -38,6 +38,9 @@ using viam::service::slam::v1::SLAMService;
 namespace viam {
 
 static const int checkForShutdownIntervalMicroseconds = 1e5;
+static const float maximumGRPCByteLimit = 3200000;
```

Comment:
[q] in main.cc `builder.SetMaxSendMessageSize(32 * 1024 * 1024);` can we change this to `builder.SetMaxSendMessageSize(maximumGRPCByteLimit);` (and maybe change 3200000 to 32 * 1024 * 1024 or 32 *2<20